pom.xml

```diff
@@ -109,8 +109,9 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <maven.version>3.8.7</maven.version>
-        <mvnd.version>0.9.0</mvnd.version>
+        <maven.version>3.9.1</maven.version>
+        <mvnd.version>1.0-m6</mvnd.version>
+        <mvnd.maven.short.version>39</mvnd.maven.short.version>
```

Comment:
we don't need it, let's evaluate it in shell script, using `maven.version`

Reply:
Thanks for the code suggestion. Applied.